### PR TITLE
Fix the response body of an error sent by MeiliSearch

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

**Comment by curquiza** [comment]
@react-learner I don't know how to fix the link... I will ask Charlotte expect if you know how to do it!

**Comment by react-learner** [comment]
@curquiza I know you can fix the link by making it an external e.g. https://docs.meilisearch.com/errors, but not sure how to link to the errors page using the VuePress notation.

File: reference/api/README.md [hunk 1]
```diff
@@ -55,10 +55,15 @@ Response body:
...
 {
-  "message": "The error message"
+  "message": "Index movies not found",
+  "errorCode": "index_not_found",
+  "errorType": "invalid_request_error",
+  "errorLink": "https://docs.meilisearch.com/errors#index_not_found"
 }
 ```
 
+The [complete list](/errors) of all the `errorCode` with their detailed description might help you understanding the MeiliSearch errors.
```
Comment (curquiza) [inline reference/api/README.md:65]:
```suggestion
The [complete list](https://docs.meilisearch.com/errors) of all the `errorCode` with their detailed description might help you understanding the MeiliSearch errors.
```

**Comment by curquiza** [comment]
Done!

**Comment by curquiza** [comment]
Feel free to change my way of writing english sentences of course, I will not be upset 😂

File: reference/api/README.md [hunk 2]
```diff
@@ -55,10 +55,15 @@ Response body:
...
 {
-  "message": "The error message"
+  "message": "Index movies not found",
+  "errorCode": "index_not_found",
+  "errorType": "invalid_request_error",
+  "errorLink": "https://docs.meilisearch.com/errors#index_not_found"
 }
 ```
 
+The [complete list](https://docs.meilisearch.com/errors) of all the `errorCode` with their detailed description might help you understanding the MeiliSearch errors.
```
Comment (react-learner) [inline reference/api/README.md:65]:
```suggestion
If you're having trouble understanding an error, take a look at the [complete list](https://docs.meilisearch.com/errors) of `errorCode` values and descriptions.
```

**Review by react-learner — APPROVED** [review]
LGTM 😄 Thanks for the update!

**Review by react-learner — APPROVED** [review]

**Pull request merged**